packages/quantic/cypress/e2e/default-2/generatedAnswer/generated-answer-actions.ts

```diff
@@ -0,0 +1,36 @@
+import {
+  GeneratedAnswerSelector,
+  GeneratedAnswerSelectors,
+} from './generated-answer-selectors';
+
+function generatedAnswerActions(selector: GeneratedAnswerSelector) {
+  return {
+    likeGeneratedAnswer: () =>
+      selector
+        .likeButton()
+        .click()
+        .logAction('When clicking on the like button of the generated answer'),
+
+    dislikeGeneratedAnswer: () =>
+      selector
+        .dislikeButton()
+        .click()
+        .logAction(
+          'When clicking on the dislike button of the generated answer'
+        ),
+
+    clickCitation: (index: number) =>
+      selector
+        .citationLink(index)
+        .then((element) => {
+          // In the tests, we want to avoid opening a citation source in a new tab, cause originally the value of the target attribute is set to "_blank"
+          element.get(0).setAttribute('target', '_self');
```

Comment:
cypress doesn't like _blank targets? :D

Reply:
No, I just don't want to open a new tab while running the tests.
 I wanna stay in the same page where I'm doing my tests, so the tests happening after this action don't fail.